librz/asm/arch/rl78/rl78.h

```diff
@@ -0,0 +1,23 @@
+// SPDX-FileCopyrightText: 2023 Bastian Engel <bastian.engel00@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#ifndef RL78_H
+#define RL78_H
+
+#include "rl78_instr.h"
+
+#include <rz_util.h>
+#include <rz_types.h>
+
+/**
```

Comment:
Move the Doxygen to the source file, e.g. see https://github.com/rizinorg/rizin/pull/3891